Lib/test/support/_hypothesis_stubs/__init__.py

```diff
@@ -0,0 +1,71 @@
+import functools
+import unittest
+
+__all__ = [
+    "given",
+    "example",
+    "assume",
+    "reject",
+    "register_random",
+    "strategies",
+    "HealthCheck",
+    "settings",
+    "Verbosity",
+]
+
+from . import strategies
+from ._settings import HealthCheck, Verbosity, settings
+from .control import assume, reject
+
+
+def given(*_args, **_kwargs):
+    def decorator(f):
+        if getattr(f, "_given", False):
+            # given can be used more than once, but for our purposes there's
+            # no need to wrap the function multiple times.
+            return f
```

Comment:
Actually, applying `@given()` multiple times is an error, so this can be taken out too.

Reply:
Ah, I didn't realize that had changed!